### ISSUE TITLE
Help with Server Issues Using Sophisticated Core and Neoforge 20.4.237

### ISSUE THREAD

**Issue by Projekt04** [P3pp3rF1y/SophisticatedCore#246]
Hello everyone,

I am trying to host a private server on my computer and I want to use the Sophisticated Mods. I am playing on the "Neoforge 20.4.237" version. The mods work perfectly in single-player mode, but they do not work on the server. I am getting an error message indicating that certain channels of the mod "Sophisticated Core" are missing on the server side but required on the client.

I have verified that both the server and the client are using the same version of Neoforge and the same set of mods. I also tried running the server with only essential mods like voice chat and cloth config, and it worked fine. However, the issue persists with the Sophisticated Backpacks mod, even though it works in single-player mode.

I have already tried using Java 17 on the server instead of a newer version, but the problem remains.

Can anyone help me resolve this issue? Where can I get in contact with the developer? Is CurseForge the best place for this?

Thank you in advance for your help!
![grafik](https://github.com/P3pp3rF1y/SophisticatedCore/assets/173090285/22c50f64-f9b4-4dd4-ab78-3d9581b8ae71)

**Comment by P3pp3rF1y** [comment]
have you got JEI on the server, this looks like its compat only loaded on client but not on server

**Comment by Projekt04** [comment]
Hi, 
I apologize for the late response; I have been very busy with work this week.

I have now installed JEI on both the server and the client as mentioned in your comment. It was indeed only installed on the client side initially. However, I am now receiving new error messages.

One of the errors claims that the Sophisticated Storage mod is only installed on the client, but I have verified that it is installed on both the server and the client. Unfortunately, I don't understand the other error messages.
Here is a screenshot of the latest errors I am encountering:

![grafik](https://github.com/P3pp3rF1y/SophisticatedCore/assets/173090285/7d4f1499-605c-4ab5-97a1-fed872aec0e8)


Could you please help me with these new issues?

**Comment by Projekt04** [comment]
Hi,
I apologize for the late response; I have been very busy with work this week.

I have now installed JEI on both the server and the client as mentioned in your comment. It was indeed only installed on the client side initially. However, I am now receiving new error messages.

One of the errors claims that the Sophisticated Storage mod is only installed on the client, but I have verified that it is installed on both the server and the client. Unfortunately, I don't understand the other error messages.
Here is a screenshot of the latest errors I am encountering:

![grafik](https://github.com/P3pp3rF1y/SophisticatedCore/assets/173090285/63369198-ec34-41f2-8db4-3074f809bb06)

Could you please help me with these new issues?

**Comment by P3pp3rF1y** [comment]
I have no info based on which I can help you. Have you checked server log to make sure that sophisticated storage gets loaded and there are no errors that prevent it from loading?

**Comment by Projekt04** [comment]
I have checked the server logs and found no issues that would prevent Sophisticated Storage from loading. To troubleshoot further, I removed the Sophisticated Storage mod from the mods folder, and the server started successfully. JEI displayed the items, indicating that the mods were working correctly.

When I added the Sophisticated Storage mod back, the error reappeared. I then re-downloaded the mod, and now the server starts with all mods installed.

It seems that this has resolved the issue. Unfortunately, I can't pinpoint the exact cause, but there might have been an issue with the previous file. But i'm not sure about that. 

I apologize for any inconvenience and greatly appreciate your support and assistance.
